plugins/Morpheus/javascripts/piwikHelper.js

```diff
@@ -23,6 +23,42 @@ function _pk_translate(translationStringId, values) {
     return "The string "+translationStringId+" was not loaded in javascript. Make sure it is added in the Translate.getClientSideTranslationKeys hook.";
 }
 
+function _pk_externalRawLink(url, values) {
+
+  if (!url) {
+    return '';
+  }
+
+  const campaignOverride = (typeof values != 'undefined' && values.length > 0 && values[0] ? values[0] : null);
+  const sourceOverride = (typeof values != 'undefined' && values.length > 1 && values[1] ? values[1] : null);
+  const mediumOverride = (typeof values != 'undefined' && values.length > 2 && values[2] ? values[2] : null);
+
+  const returnURL = new URL(url);
+  const validDomain = returnURL.host.endsWith('matomo.org');
```

Comment:
to be on the safe side with subdomains and not just catch `anyrandomdomainformatomo.org`
```suggestion
  const validDomain = returnURL.host === 'matomo.org' || returnURL.host.endsWith('.matomo.org');
```